bob-common/src/metrics/exporters/global_exporter/rate_processor.rs

```diff
@@ -0,0 +1,65 @@
+const BUFFER_SIZE: usize = 10_485_760; // 10 Mb
```

Comment:
Just note, it is not 10Mb, `memory_usage = BUFFER_SIZE * size_of(Message)`

Reply:
Yes, that's incorrect, it's size in messages